packages/examples/bike_simulator.py

```diff
@@ -38,9 +38,14 @@
 
 class PackageClass(LiotaPackage):
 
-    def run(self, registry):
+    def run(self, registry, package_record):
         from liota.entities.devices.bike_simulated import BikeSimulated
         import pint
+        from liota.lib.utilities.utility import check_integrity
+
+        # verify file integrity first
+        if (check_integrity(package_record, __file__) == False):
```

Comment:
Sorry that after discussed with Greg, I agreed with you and will modify codes to let user pass in checksum.